### ISSUE TITLE
Opacity for Mesh Geometries

### ISSUE THREAD

**Issue by mturiansky** [glotzerlab/fresnel#91]
## Description

It would be nice to be able to specify an RGBA value for the color of meshes.

## Preview

I'd like to reproduce the isosurfaces with the opacity shown in (b) and (c) of this figure using fresnel:

![image](https://journals.aps.org/prl/article/10.1103/PhysRevLett.123.127401/figures/1/medium)

## Developer

I'd be happy to look into implementing this if someone could push me in the right direction.

**Comment by tcmoore3** [comment]
@mturiansky is the `spec_trans` material property not what you are looking for?

**Comment by mturiansky** [comment]
@tcmoore3 Oh wow, now I feel dumb. That is exactly what I'm looking for. The spec_trans property just doesn't work in the interactive SceneView, right?

**Comment by tcmoore3** [comment]
@mturiansky I think it should work in a SceneView, but Qt widgets are notoriously finicky, so there may be something weird going on with different versions of different libraries.

**Comment by joaander** [comment]
`spec_trans` should work in the interactive scene view. Maybe you are using the _Preview_ tracer? `spec_trans` only works with the [Path tracer](https://fresnel.readthedocs.io/en/stable/module-tracer.html#fresnel.tracer.Path). That should be documented somewhere, but it is not. Also, there needs to be light coming from behind the object to pass through the material, so it works best with `cloudy` or `lightbox` lighting. There should also be an example dedicated to `spec_trans`, specifically covering the lighting issues (which are not obvious). 

I am reopening this as a reminder to add this documentation.

**Comment by mturiansky** [comment]
You're correct; `spec_trans` does work in the interactive scene view. I think the combination of sub-optimal lighting, lower sampling, and relatively low `spec_trans` value made the transparency not obvious. It wasn't until I generated a static image with good sampling that I could see the transparency.
